.github/workflows/tests_linux_x86.yml

```diff
@@ -131,7 +131,7 @@ jobs:
 
       - name: Build and install package
         run: |
-            python setup.py build_ext -i --define="CMAKE_CXX_COMPILER=$(which g++-${{ env.GCC_VERSION }});LIGHTNING_RELEASE_TAG=master;CMAKE_CUDA_COMPILER=/usr/local/cuda/bin/nvcc;CMAKE_CUDA_ARCHITECTURES=${{ env.CI_CUDA_ARCH }}"
```

Comment:
Why do we have `LIGHTNING_RELEASE_TAG=latest_release` here but `LIGHTNING_RELEASE_TAG=master` elsewhere?

Reply:
Because I am foolish and forgot to do the update ;) 
This is why reviews are so helpful. Will make the change tomorrow before merging in.

Reply:
Everything updated to `master` now. Thanks @vincentmr